demo/src/profile.ts

```diff
@@ -0,0 +1,70 @@
+import * as Cord from '@cord.network/sdk';
+
+async function main() {
+  const networkAddress = process.env.NETWORK_ADDRESS || 'ws://127.0.0.1:9944';
+
+  try {
+    // 🏦 Network Connection
+    console.log(`\n🏦 Connecting to CORD network at ${networkAddress}...`);
+    Cord.ConfigService.set({ submitTxResolveOn: Cord.Chain.IS_IN_BLOCK });
+    await Cord.connect(networkAddress);
+
+    const api = Cord.ConfigService.get('api');
+    const runtimeVersion = api.runtimeVersion;
+    const runtimeType = runtimeVersion.specName.toString();
+    console.log(`✅ Connected to CORD runtime: ${runtimeType} (version ${runtimeVersion.specVersion})`);
+
+    // 👤 Setup Network Members
+    console.log('\n👤 Setting up network members...');
+    const alice = Cord.Utils.Crypto.makeKeypairFromUri(
+      process.env.ANCHOR_URI || '//Alice',
+      'sr25519'
+    );
+    const bob = Cord.Utils.Crypto.makeKeypairFromUri(
+      process.env.ANCHOR_URI || '//Bob',
+      'sr25519'
+    );
+    console.log(`🏦 Member 1 (Alice): ${alice.address}`);
+    console.log(`🏦 Member 2 (Bob): ${bob.address}`);
+
+    // 📝 Create Profile for Alice
+    console.log(`\n📝 Creating profile for Alice (${alice.address})...`);
+
+    // 🔑 Set Profile Data
+    const jsonProfileData = { "pub_name": "Alice", "pub_email": "alice@example.com" };
```

Comment:
Can you give example as 'hash' of email/name ? Just to give an idea for users/developers that this is hash we 'recommend'.

(while it is true that they can set anything).

Reply:
Sure! That would be good.